Description: Set defaults that make sense for Debian systems.
Index: prometheus-node-exporter/collector/netdev_common.go
===================================================================
--- prometheus-node-exporter.orig/collector/netdev_common.go
+++ prometheus-node-exporter/collector/netdev_common.go
@@ -26,7 +26,7 @@ import (
 )
 
 var (
-	netdevIgnoredDevices = kingpin.Flag("collector.netdev.ignored-devices", "Regexp of net devices to ignore for netdev collector.").Default("^$").String()
+	netdevIgnoredDevices = kingpin.Flag("collector.netdev.ignored-devices", "Regexp of net devices to ignore for netdev collector.").Default("^lo$").String()
 )
 
 type netDevCollector struct {
Index: prometheus-node-exporter/collector/filesystem_linux.go
===================================================================
--- prometheus-node-exporter.orig/collector/filesystem_linux.go
+++ prometheus-node-exporter/collector/filesystem_linux.go
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	defIgnoredMountPoints = "^/(dev|proc|sys|var/lib/docker)($|/)"
+	defIgnoredMountPoints = "^/(dev|proc|run|sys|mnt|media|var/lib/docker)($|/)"
 	defIgnoredFSTypes     = "^(autofs|binfmt_misc|cgroup|configfs|debugfs|devpts|devtmpfs|fusectl|hugetlbfs|mqueue|overlay|proc|procfs|pstore|rpc_pipefs|securityfs|sysfs|tracefs)$"
 	readOnly              = 0x1 // ST_RDONLY
 )
Index: prometheus-node-exporter/collector/textfile.go
===================================================================
--- prometheus-node-exporter.orig/collector/textfile.go
+++ prometheus-node-exporter/collector/textfile.go
@@ -32,7 +32,7 @@ import (
 )
 
 var (
-	textFileDirectory = kingpin.Flag("collector.textfile.directory", "Directory to read text files with metrics from.").Default("").String()
+	textFileDirectory = kingpin.Flag("collector.textfile.directory", "Directory to read text files with metrics from.").Default("/var/lib/prometheus/node-exporter").String()
 	mtimeDesc         = prometheus.NewDesc(
 		"node_textfile_mtime_seconds",
 		"Unixtime mtime of textfiles successfully read.",
Index: prometheus-node-exporter/collector/systemd_linux.go
===================================================================
--- prometheus-node-exporter.orig/collector/systemd_linux.go
+++ prometheus-node-exporter/collector/systemd_linux.go
@@ -28,7 +28,7 @@ import (
 
 var (
 	unitWhitelist  = kingpin.Flag("collector.systemd.unit-whitelist", "Regexp of systemd units to whitelist. Units must both match whitelist and not match blacklist to be included.").Default(".+").String()
-	unitBlacklist  = kingpin.Flag("collector.systemd.unit-blacklist", "Regexp of systemd units to blacklist. Units must both match whitelist and not match blacklist to be included.").Default(".+\\.scope").String()
+	unitBlacklist  = kingpin.Flag("collector.systemd.unit-blacklist", "Regexp of systemd units to blacklist. Units must both match whitelist and not match blacklist to be included.").Default(".+(\\.device|\\.scope|\\.slice|\\.target)").String()
 	systemdPrivate = kingpin.Flag("collector.systemd.private", "Establish a private, direct connection to systemd without dbus.").Bool()
 )
 
@@ -44,7 +44,7 @@ type systemdCollector struct {
 var unitStatesName = []string{"active", "activating", "deactivating", "inactive", "failed"}
 
 func init() {
-	registerCollector("systemd", defaultDisabled, NewSystemdCollector)
+	registerCollector("systemd", defaultEnabled, NewSystemdCollector)
 }
 
 // NewSystemdCollector returns a new Collector exposing systemd statistics.
